app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -3,51 +3,50 @@
   <% @seen_summary = {} %>
   <% @creations.each_with_index do |creation, index| %>
 
-    <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
+<% this_work = creation.respond_to?(:work) ? creation.work : creation %>
```

Comment:
Since this pull request needs to be updated before it can be merged, a few comments! The indenting has gone a bit wonky here from here down to `<% unless @seen[this_work.id] %>`. I believe what you want is:

```
    <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
    <% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
    <% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
    <% title = this_work.title.html_safe %>
    <%
      if creation.is_a?(Work) 
        link = style_link(this_work.title.html_safe,work_url(this_work))
        word_count = this_work.word_count
      else
        link = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
        chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
      end
    %>
    <% if creation.is_a?(Work) %>
      <%= 
        if this_work.backdate
          t('user_mailer.batch_subscription_notification.html.work_backdated', byline: byline, title: title, link: link, word_count: word_count).html_safe
        else
          t('user_mailer.batch_subscription_notification.html.work_new', byline: byline, title: title, link: link, word_count: word_count).html_safe
        end
      %>
    <% else %>
      <%= t('user_mailer.batch_subscription_notification.html.new_chapter_of', byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title).html_safe %>
    <% end %>
    <%= t('user_mailer.batch_subscription_notification.html.collected_by', pseud: this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe).html_safe %>

    <% if !creation.summary.blank? && creation.is_a?(Chapter) %>
      <%= t 'user_mailer.batch_subscription_notification.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
    <% end %>
```

There are various other fixes in what I just pasted, but I'm going to point them out below as well, so you can see where they were.